File: Documents/NetBeansProjects/Web_java/src/web_java/forum.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package web_java;
import java.util.Date;
/**
 *
 * @author DELL
 */
   public class forum {
   private long id_forum;
   private String date_creation_forum;
   private long id_sujet_forum;
   private long id_user_admin;
   private String liste_participants;

    public forum(long id_forum, String date_creation_forum, long id_sujet_forum, long id_user_admin, String liste_participants) {
        this.id_forum = id_forum;
        this.date_creation_forum = date_creation_forum;
        this.id_sujet_forum = id_sujet_forum;
        this.id_user_admin = id_user_admin;
        this.liste_participants = liste_participants;
    }

    public forum(String date_creation_forum, long id_sujet_forum, long id_user_admin, String liste_participants) {
        this.date_creation_forum = date_creation_forum;
        this.id_sujet_forum = id_sujet_forum;
        this.id_user_admin = id_user_admin;
        this.liste_participants = liste_participants;
    }

    public forum() {
    }

    public long getId_forum() {
        return id_forum;
    }

    public void setId_forum(long id_forum) {
        this.id_forum = id_forum;
    }

    public String getDate_creation_forum() {
        return date_creation_forum;
    }

    public void setDate_creation_forum(String date_creation_forum) {
        this.date_creation_forum = date_creation_forum;
    }

    public long getId_sujet_forum() {
        return id_sujet_forum;
    }

    public void setId_sujet_forum(long id_sujet_forum) {
        this.id_sujet_forum = id_sujet_forum;
    }

    public long getId_user_admin() {
        return id_user_admin;
    }

    public void setId_user_admin(long id_user_admin) {
        this.id_user_admin = id_user_admin;
    }

    public String getListe_participants() {
        return liste_participants;
    }

    public void setListe_participants(String liste_participants) {
        this.liste_participants = liste_participants;
    }

    @Override
    public String toString() {
        return "forum{" + "date_creation_forum=" + date_creation_forum + ", id_sujet_forum=" + id_sujet_forum + ", id_user_admin=" + id_user_admin + ", liste_participants=" + liste_participants + '}';
    }
   
   
   
   

   
}

File: mysymfony2/IRON_FIT_Front-PHP-Symfony-main/src/Entity/Forum.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use App\Repository\ForumRepository;


/**
 * @ORM\Entity
 * @ORM\Table(name="forum")
 */
#[ORM\Entity]
#[ORM\Table(name: "forum")]
class Forum
{
    #[ORM\Id]
    #[ORM\GeneratedValue(strategy: "AUTO")]
    #[ORM\Column(type: "bigint")]
    private $idForum;

    #[ORM\Column(type: "string", length: 255)]
    private $dateCreationForum;

    #[ORM\Column(type: "bigint")]
    private $idSujetForum;

    #[ORM\Column(type: "bigint")]
    private $idUserAdmin;

    #[ORM\Column(type: "text", nullable: true)]
    private $listeParticipants;

    // Getters and Setters

    public function getIdForum(): ?int
    {
        return $this->idForum;
    }

    public function setIdForum(?int $idForum): self
    {
        $this->idForum = $idForum;
        return $this;
    }

    public function getDateCreationForum(): ?string
    {
        return $this->dateCreationForum;
    }

    public function setDateCreationForum(string $dateCreationForum): self
    {
        $this->dateCreationForum = $dateCreationForum;
        return $this;
    }

    public function getIdSujetForum(): ?string
    {
        return $this->idSujetForum;
    }

    public function setIdSujetForum(string $idSujetForum): self
    {
        $this->idSujetForum = $idSujetForum;
        return $this;
    }

    public function getIdUserAdmin(): ?string
    {
        return $this->idUserAdmin;
    }

    public function setIdUserAdmin(string $idUserAdmin): self
    {
        $this->idUserAdmin = $idUserAdmin;
        return $this;
    }

    public function getListeParticipants(): ?string
    {
        return $this->listeParticipants;
    }

    public function setListeParticipants(?string $listeParticipants): self
    {
        $this->listeParticipants = $listeParticipants;
        return $this;
    }


    // __toString() method
    public function __toString(): string
    {
        return sprintf(
            'Forum ID: %d, Created on: %s',
            $this->getIdForum(),
            $this->getDateCreationForum() ?? 'N/A'
        );
    }
}
